src/prefabs/structures/Box/options/index.ts

```diff
@@ -145,6 +147,13 @@ export const boxOptions = {
       },
     },
   ),
+  zIndex: number('Z-index', {
+    value: '0',
+    configuration: {
+      condition: showIf('position', 'EQ', 'fixed'),
```

Comment:
I think this condition should be this:
```typescript
condition: showIf('position', 'NEQ', 'static'),
```
You want to always show the zIndex, except if the position is set to 'static'

Reply:
@dylanBetty appearently NEQ doesn't exists, therefore I choose the option to use 
`condition: hideIf('position', 'EQ', 'static')`